samples/ios/app/glean-sample-app/AppDelegate.swift

```diff
@@ -13,7 +13,16 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         let glean = Glean.shared
         glean.setUploadEnabled(true)
-        glean.initialize()
+
+        if ProcessInfo.processInfo.arguments.contains("USE_MOCK_SERVER") {
+            let address = "http://localhost:9080"
+            print("using a mock server, setting address: \(address)")
```

Comment:
Is this and the one below meant to be a `printf`? Does Apple use any other logging mechanism that we should use here?

Reply:
`print` it is! (or `NSLog`, which is slower).  We use the same in the Logger class in Glean.

As this is just the sample app I didn't want to add more sophisticated logging.